lib/nexmo_developer/Gemfile

```diff
@@ -129,7 +129,7 @@ gem 'lograge'
 gem 'countries'
 gem 'country_select', '~> 4.0'
 
-gem 'nexmo-oas-renderer', '~> 2.4.1', require: false
+gem 'nexmo-oas-renderer', github: 'nexmo/nexmo-oas-renderer', branch: 'add-title-to-switcher', require: false
```

Comment:
I think the PR on this branch was merged and released?

Reply:
I'll check whether this was released or not and update it accordingly 👍